tests/integration/src/xvm.rs

```diff
@@ -157,6 +157,9 @@ const CALL_EVM_PAYBLE_NAME: &'static str = "call_xvm_payable";
 #[test]
 fn evm_payable_call_via_xvm_works() {
     new_test_ext().execute_with(|| {
+        // create account mappings
+        claim_default_accounts(ALICE);
```

Comment:
All it to `new_test_ext` to avoid repeating everywhere.

Reply:
https://github.com/AstarNetwork/Astar/pull/1019#discussion_r1326990724